app/src/main/res/raw/b2c_config.json

```diff
@@ -0,0 +1,22 @@
+{
+  "client_id" : "90c0fe63-bcf2-44d5-8fb7-b8bbc0b29dc6",
+  "authorization_user_agent" : "DEFAULT",
+  "redirect_uri" : "msal90c0fe63-bcf2-44d5-8fb7-b8bbc0b29dc6://auth",
+  "authorities" : [
+    {
+      "type": "B2C",
+      "authority_url": "https://login.microsoftonline.com/tfp/fabrikamb2c.onmicrosoft.com/B2C_1_SUSI",
```

Comment:
recommend using *.b2clogin.com

Reply:
@valnav who owns this app registration? We would need to update all the samples, as we all share the same info for the samples, but it would be nice to reflect *.b2clogin.com and not login.msonline.com...Can we have both registered, so this wouldn't break other samples? Let us know if/when you update so we can update the .NET samples as well. Would be awesome!
cc: @parakhj @jmprieur